static/js/hotkey.js

```diff
@@ -19,6 +19,9 @@ var actions_dropdown_hotkeys = [
 // we'll do in cases where they have the exact same semantics.
 // DON'T FORGET: update keyboard_shortcuts.html
 
+// message_view_only is a boolean which is true if the keyboard 
+// shortcut works only when the message view is in focus
```

Comment:
This really does not do much explaining right?

>  works only when the message view is in focus?

I don't think a new comer will be able to understand this just by reading it. Can you explain it in very rudimentary terms so that anyone reading it for the first time can understand it?